### PR TITLE
Y'know what fuck you *alphabetizes your menus*

### PR DESCRIPTION
**Pull request by Cartlord**
# General Documentation

### Intent of your Pull Request

I went though every sheet crafting menu, and alphabetized them. The only one I didn't adjust was plastic, for the sake of having "large water bottle" adjacent to "water bottle".

### Why is this change good for the game?
Makes it easier to find stuff if you're not already familiar with the sheet menus, will mess with muscle memory for those of us used to what there is now but will likely be easier to get used to.

### Briefly describe your PR and the impacts of it, in layman's terms. 
When you click or press Z to use sheets (e.g. iron sheets, wood planks), the menu will be ordered differently.

### What should players be aware of when it comes to the changes your PR is implementing?
It'll be a bit annoying to make stuff if you're used to how the menus are organized now, but this should be somewhat easy to get used to (I hope).

### What general grouping does this PR fall under? 
Menu tweak

# Changelog

:cl:  
tweak: Alphabetized the menus that show up when you use material sheets in-hand
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
for the record i didn't actually test this but i also see no reason it'd be any issue

**Comment by Cartlord** [comment]
I, genuinely, have no idea why the checks are failing.

**Comment by adamsong** [comment]
You need to check the annotate linter section for more details
```
code/game/objects/items/stacks/sheets/sheet_types.dm, line 18, column 1:
error: got 'new', expected one of: operator, field access, ')', ','

code/game/objects/items/stacks/sheets/sheet_types.dm, line 397, column 1:
error: got 'new', expected one of: operator, field access, ')', ','
```

**Comment by Cartlord** [comment]
Terribly sorry about that.

**Comment by Cartlord** [comment]
Closing until I can figure this out.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -15,40 +15,40 @@
...
 		new/datum/stack_recipe("lime comfy chair", /obj/structure/chair/comfy/lime, 2, one_per_turf = TRUE, on_floor = TRUE), \
 		new/datum/stack_recipe("teal comfy chair", /obj/structure/chair/comfy/teal, 2, one_per_turf = TRUE, on_floor = TRUE), \
 		)), \
+	new/datum/stack_recipe_list("office chairs", list( \
+		new/datum/stack_recipe("dark office chair", /obj/structure/chair/office/dark, 5, one_per_turf = TRUE, on_floor = TRUE), \
+		new/datum/stack_recipe("light office chair", /obj/structure/chair/office/light, 5, one_per_turf = TRUE, on_floor = TRUE), \
+		)), \
 	new/datum/stack_recipe_list("sofas", list(
-		new /datum/stack_recipe("sofa (middle)", /obj/structure/chair/sofa, 1, one_per_turf = TRUE, on_floor = TRUE),
+		new /datum/stack_recipe("sofa (corner)", /obj/structure/chair/sofa/corner, 1, one_per_turf = TRUE, on_floor = TRUE)
```
Comment (Asdjky) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: There's a comma missing at the end of this line.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -394,50 +394,50 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
...
-		new /datum/stack_recipe("sterile masks box", /obj/item/storage/box/masks),				\
 		new /datum/stack_recipe("body bag box", /obj/item/storage/box/bodybags),					\
+		new /datum/stack_recipe("latex gloves box", /obj/item/storage/box/gloves),				\
+		new /datum/stack_recipe("pillbottle box", /obj/item/storage/box/pillbottles),			\
 		new /datum/stack_recipe("prescription glasses box", /obj/item/storage/box/rxglasses),	\
+		new /datum/stack_recipe("sterile masks box", /obj/item/storage/box/masks),				\
+		new /datum/stack_recipe("syringe box", /obj/item/storage/box/syringes),					\
 		null,																					\
 
+		new /datum/stack_recipe("candle box", /obj/item/storage/box/fancy/candle_box)			\
```
Comment (Asdjky) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: There's a comma missing at the end of this line.

**Comment by Cartlord** [comment]
Thanks to what Asdjky showed me, I _think_ I got this working. I'm not 100% sure how PRs work, so I might be wrong here, and I still haven't tested this - but I added commas where pointed out.

**Review by adamsong — APPROVED** [review]
Tested and working as described

**Comment by Cartlord** [comment]
based adam
